app/hotels/src/map/HotelDetailPreview.js

```diff
@@ -37,21 +37,29 @@ const styles = StyleSheet.create({
   description: {
     flex: 1,
     justifyContent: 'space-between',
-    marginVertical: 3,
   },
   hotelName: {
-    fontSize: 16,
     color: '#30363d',
+    fontSize: 17,
+    android: {
+      lineHeight: 16,
```

Comment:
I think this is no go. Line height cannot be smaller than font size on Android otherwise the characters are going to be cropped (characters like "ěščřžýáíé"). I was thinking about extending our PlatformStyleSheet with validation but we are still using this for +- buttons hack...

Same even for other lines.

Reply:
I can also set line height higher, this was just the values I found in Zeplin.

Let's see if we can find some better solution anyways...

Reply:
I am ok with line-height solution just not with the number. Zeplin design is not very RN compatible... 🙂

Reply:
See: https://trello.com/c/S69utn2g/143-hotel-name-on-detail or https://trello.com/c/ObSFWfrs/149-bug-description-string-in-search-is-not-fully-visible

Reply:
Ok, let me have a second look at it when they have fixed the maven issue, now I can't run the app on android...